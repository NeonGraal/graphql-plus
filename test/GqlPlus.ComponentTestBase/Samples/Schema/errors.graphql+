category query {}
directive @ {}
domain Other { Enum }
enum None [] {}
union Missing {}
dual Nothing [] {}
input Empty {}
output NoParams<> {}
