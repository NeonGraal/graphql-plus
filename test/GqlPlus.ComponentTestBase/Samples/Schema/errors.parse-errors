Invalid Category. Expected output type.
Invalid Schema. Expected no more text.