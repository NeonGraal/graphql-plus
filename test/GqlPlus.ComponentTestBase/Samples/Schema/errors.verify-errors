Invalid Category Output. '' not defined or not an Output type.