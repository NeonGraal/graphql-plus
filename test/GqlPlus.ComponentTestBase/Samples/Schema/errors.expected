Invalid Category Output. '' not defined or not an Output type.
Invalid Category. Expected output type.
Invalid Schema. Expected no more text.